core/storage/user/gae_models_test.py

```diff
@@ -2276,12 +2276,17 @@ def test_create_raises_error_when_many_id_collisions_occur(self):
 class UserAuthModelTests(test_utils.GenericTestBase):
     """Tests for UserAuthModel."""
 
-    NONEXISTENT_AUTH_TYPE_NAME = 'id_x'
+    NONEXISTENT_AUTH_METHOD_NAME = 'id_x'
```

Comment:
Why call this "id_..."? Should be "auth_method_x" perhaps?

Reply:
Sorry I hadn't paid attention to it earlier. Changed now.

Reply:
This doesnt seem changed?

Reply:
Sorry fixed it now.